### PR TITLE
Feature/7.x/low level expose deprecated paths

### PR DESCRIPTION
**Pull request by Mpdreamz**
Adds deprecated `paths` back as exposed in the low level client.

* Gives users the control to keep using it if they must (remember you can reuse NEST response types)
* As API's get deprecated over minors we will be ready for it
* Gives us the ability to reimplement the YAML tests during 7.0's lifetime

### CODE REVIEW

File: src/CodeGeneration/ApiGenerator/Domain/Specification/UrlInformation.cs [hunk 1]
```diff
@@ -27,6 +34,47 @@ public IReadOnlyCollection<UrlPath> Paths
...
 		}
+		
+		private List<UrlPath> _pathsWithDeprecation;
+		public IReadOnlyCollection<UrlPath> PathsWithDeprecations
+		{
+			get
+			{
+				if (Paths.Any(p => p.Path.Contains("plugins")))
+				{
+
```
Comment (codebrain) [inline src/CodeGeneration/ApiGenerator/Domain/Specification/UrlInformation.cs]: noop intentional?

File: src/CodeGeneration/ApiGenerator/Domain/Specification/UrlPart.cs [hunk 2]
```diff
@@ -2,33 +2,45 @@
...
+
+		public string LowLevelTypeName
 		{
 			get
 			{
 				//TODO treat list with fixed options as Flags Enum
 				switch (Type)
 				{
-					case "int":
+					case "int": //last i checked does not occur on part
```
Comment (codebrain) [inline src/CodeGeneration/ApiGenerator/Domain/Specification/UrlPart.cs]: Shorten to "Does not occur on part" as I started hunting for a variable named `i` :)

File: src/CodeGeneration/ApiGenerator/Views/LowLevel/Client/Methods/MethodDocs.cshtml [hunk 3]
```diff
@@ -16,7 +16,7 @@
 </text>
 }
 		///@Raw(@"<param name=""requestParameters"">Request specific configuration such as querystring parameters &amp; request specific connection settings.</param>")
-		@if (!string.IsNullOrEmpty(method.DeprecatedPath))
+		@if (method.DeprecatedPath != null)
 		{
-<text>		[Obsolete("@method.DeprecatedPath")]
+<text>		[Obsolete("Deprecated as of @(method.DeprecatedPath.Version): @Raw(method.DeprecatedPath.Description)")]
```
Comment (codebrain) [inline src/CodeGeneration/ApiGenerator/Views/LowLevel/Client/Methods/MethodDocs.cshtml]: Maybe

[Obsolete("Deprecated in version @(method.DeprecatedPath.Version): @Raw(method.DeprecatedPath.Description)")]

File: src/CodeGeneration/ApiGenerator/Domain/Specification/UrlPart.cs (hunk 2 above)
Comment (Mpdreamz) [reply src/CodeGeneration/ApiGenerator/Domain/Specification/UrlPart.cs]: :joy_cat: sorry about that updated

File: src/CodeGeneration/ApiGenerator/Domain/Specification/UrlInformation.cs (hunk 1 above)
Comment (Mpdreamz) [reply src/CodeGeneration/ApiGenerator/Domain/Specification/UrlInformation.cs]: very much not so :crying_cat_face:

File: src/CodeGeneration/ApiGenerator/Views/LowLevel/Client/Methods/MethodDocs.cshtml (hunk 3 above)
Comment (Mpdreamz) [reply src/CodeGeneration/ApiGenerator/Views/LowLevel/Client/Methods/MethodDocs.cshtml]: Updated.

**Comment by Mpdreamz** [comment]
Integration test failures unrelated will follow up.

**Pull request merged**